src/inference/src/cache_manager.hpp

```diff
@@ -111,6 +128,14 @@ class FileStorageCacheManager final : public ICacheManager {
         }
     }
 
+    void read_cache_entry(const std::string& id, BufferReader reader) override {
+        auto blob_file_name = getBlobFile(id);
+        if (ov::util::file_exists(blob_file_name)) {
+            auto buffer = ov::load_mmap_object(blob_file_name);
```

Comment:
I presume this may allocate a large blob buffer. Consider using madvise (on Linux) in order to enable transparent hugepages (THP) or anticipating sequential memory reading for a performance boost. However I am not sure if this should be implemented here or in a lower layer